lib/src/main/java/uk/gov/di/ipv/cri/address/library/persistence/item/AddressItem.java

```diff
@@ -23,6 +25,22 @@ public void setSessionId(UUID sessionId) {
         this.sessionId = sessionId;
     }
 
+    public long getCreatedDate() {
```

Comment:
Out of interest what do we need the createdDate for?

Reply:
Similar to the above I followed an existing implementation, I presumed it may be best practice incase there are any issues with the TTL. 

You're probably right though, no real need for us to have it here for such a short TTL.

Reply:
Ok cool fine to leave it in I think 👍